application/app/views/projects/_project_summary.html.erb

```diff
@@ -28,17 +28,17 @@
     <!-- Tabs -->
     <ul class="nav nav-tabs" id="tabs-<%= project.id %>" role="tablist">
       <li class="nav-item" role="presentation">
-        <a class="nav-link text-secondary active" id="download-tab" data-bs-toggle="tab"
-           href="#download-files-<%= project.id %>" role="tab"
-           aria-controls="download-files-<%= project.id %>" aria-selected="true">
+        <a class="nav-link text-secondary active" id="<%= tab_label_for(project) %>" data-bs-toggle="tab"
+           href="<%= tab_href_for(project) %>" role="tab"
+           aria-controls="<%= tab_anchor_for(project) %>" aria-selected="true">
```

Comment:
In this view we have projects and bundles together using the tab helpers. See this comment https://github.com/IQSS/ondemand-loop/pull/122/files#r2126968168